src/libraries/System.Security.Cryptography/src/System.Security.Cryptography.csproj

```diff
@@ -861,6 +1823,10 @@
     <Reference Include="System.Runtime.Numerics" />
     <Reference Include="System.Text.Encoding.Extensions" />
     <Reference Include="System.Threading" />
+    <Reference Include="System.Threading.Thread" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+    <Reference Include="System.Console" Condition="'$(Configuration)' == 'Debug'" />
```

Comment:
Is this something you intended to check in? I don't see `System.Console` used anywhere.

Reply:
Yeah, there's a `#if DEBUG` with a runtime-e-var-check trace in SafeX509Handle to look for leaks.

Copied it from the X509 csproj

Reply:
Ah, I forgot to look in `Common`.